src/css/main.less

```diff
@@ -1295,6 +1295,12 @@ dialog {
     &:hover {
         background-color: var(--primary-400);
     }
+    &.disabled {
```

Comment:
```suggestion
    &:disabled {
```

Reply:
The way it's currently handled is with a class, not a prop. Might be a good fix for a future PR